mongodbatlas/cloud_provider_access.go

```diff
@@ -40,19 +40,24 @@ var _ CloudProviderAccessService = &CloudProviderAccessServiceOp{}
 
 // CloudProviderAccessRoles an array of awsIamRoles objects.
 type CloudProviderAccessRoles struct {
-	AWSIAMRoles []AWSIAMRole `json:"awsIamRoles,omitempty"` // Unique identifier of AWS security group in this access list entry.
+	AWSIAMRoles []IAMRole `json:"awsIamRoles,omitempty"` // Unique identifier of AWS security group in this access list entry.
 }
 
-// AWSIAMRole is the response from the CloudProviderAccessService.ListRoles.
-type AWSIAMRole struct {
+// IAMRole is the response from the CloudProviderAccessService.ListRoles.
+type IAMRole struct {
```

Comment:
what about 
```suggestion
// CloudProviderAccessRole is the response from the CloudProviderAccessService.ListRoles.
type CloudProviderAccessRole struct {
```
so it matches `CloudProviderAccessRoleRequest`

Reply:
updated